app/models/customization.rb

```diff
@@ -0,0 +1,11 @@
+# typed: strict
+class Customization < ApplicationRecord
+  belongs_to :item
+
+  # Define the attributes with their respective data types and constraints
+  attribute :id, :integer
+  attribute :created_at, :datetime
+  attribute :updated_at, :datetime
+  attribute :customization_details, :text
+  attribute :item_id, :integer
+end
```

Comment:
Ensure that the 'customization_details' attribute is properly sanitized before saving to prevent potential security vulnerabilities such as XSS attacks.